Cubical/Algebra/Group/Abelianization/AbelianizationAsCoeq.agda

```diff
@@ -0,0 +1,354 @@
+{-
+
+This file contains:
+    - the abelianization of groups as a coequalizer of sets as performed
+      in https://1lab.dev/Algebra.Group.Ab.Free.html
+    - the proof that this way of defining the abelianization of groups is equivalent to defining
+      it as a HIT, more precisely that there is a unique isomorphism between the resulting
+      abelian groups
+
+-}
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.Group.Abelianization.AbelianizationAsCoeq where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+
+open import Cubical.Data.Sigma
+
+open import Cubical.Algebra.Group.Base
+open import Cubical.Algebra.Group.Properties
+open import Cubical.Algebra.Group.Morphisms
+open import Cubical.Algebra.Group.MorphismProperties
+  using (isPropIsGroupHom; compGroupHom; idGroupHom; compGroupHomAssoc; invGroupIso;
+         compGroupHomId; GroupIso≡; GroupIso→GroupHom)
+
+open import Cubical.Algebra.AbGroup.Base
+
+open import Cubical.HITs.SetCoequalizer hiding (inducedHom; commutativity; uniqueness)
+
+open import Cubical.Algebra.Group.Abelianization.Base
+  renaming (Abelianization to HITAbelianization;
+            η to HITη;
+            comm to HITcomm;
+            isset to HITisset)
+
+open import Cubical.Algebra.Group.Abelianization.Properties
+  renaming (asAbelianGroup to HITasAbelianGroup;
+            ηAsGroupHom to HITηAsGroupHom;
+            inducedHom to HITinducedHom;
+            commutativity to HITcommutativity;
+            uniqueness to HITuniqueness)
+  hiding (elimProp; elimProp2; elimProp3; elimContr; elimContr2; rec; rec2; _·Ab_; 1Ab; invAb;
+          assocAb; ridAb; rinvAb; commAb)
+
+private
+  variable
+    ℓ : Level
+    G : Group ℓ
+
+module _ (G : Group ℓ) where
```

Comment:
You should give that module a name, otherwise all its definitions will be exported. You can then, after the module, make a definition like ```AbelianizationCoeq = MODULENAME.Abelianization``` to export this definition.

Reply:
done